librz/socket/run.c

```diff
@@ -1217,7 +1217,7 @@ RZ_API int rz_run_start(RzRunProfile *p) {
 #endif
 
 		if (p->_nice) {
-#if __UNIX__ && !defined(__HAIKU__)
```

Comment:
Is it possible to define whether this functionality is available with meson's [`compiler.run`](https://mesonbuild.com/Compiler-properties.html) during compile time? Seeing that this is only available in a beta build and some people may still use older versions.

Reply:
you re kind of forced to update to beta3 via the package manager.

Reply:
We want to do that in the future. In meson.build you can check if nice function exists on the host target and then have a define HAVE_NICE If it is.

Reply:
sorry I misread the line commented, yes sure doing it.